Pid.hpp

```diff
@@ -15,11 +17,20 @@ class Pid {
     /** clear any windup */
     void clearWindup();
 
+
+    //tuner stuff
+    void initialize_tuner();
+
+    void set_from_tuner();
+
+    void start_cycle();
```

Comment:
pid tuner can't take in the pid because python can't handle pointers so it would take a lot of extra work to get it working from the play side. I will work on changing the names to be a little more descriptive.